src/content/about/metrics.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Metrics
+weight: 4
+---
+
+- [Worker Script Metrics](#worker-script-metrics)
+  - [Invocation statuses](#invocation-statuses)
+  - [Requests](#requests)
+  - [CPU time](#cpu-time)
+- [Zone Metrics](#zone-metrics)
+  - [Total requests](#total-requests)
+  - [Status codes](#status-codes)
+  - [Bandwidth](#bandwidth)
+  - [Subrequests](#subrequests)
+- [GraphQL](#graphql)
+
+Workers metrics show performance and usage of your Workers and help diagnose issues.
+
+Workers Metrics are available in Zone and Worker script contexts.
+
+## Worker Script Metrics
+
+Aggregates request data for an individual Worker script across all zones including your workers.dev subdomain. On your [Workers dashboard](https://dash.cloudflare.com/?account=workers/overview), click on any Worker to view its metrics.
+
+Worker script metrics can be inspected by time ranges within the last 30 days. The dashboard includes the charts and information described below.
+
+### Invocation statuses
+
+Worker invocation statuses are generated by the Workers runtime and indicate whether a Worker script executed successfully or failed to generate a response. Invocation statuses differ from HTTP status codes. In some cases, a Worker script invocation succeeds but does not generate a successful HTTP status because of another error encountered outside of the Workers runtime. Some invocation statuses result in a [Workers error code](/about/tips/debugging/#error-pages-generated-by-workers) being returned to the client.
+
+Worker invocation statuses:
+
+| Invocation status      | GraphQL field        | Workers Error code | Definition                                                               |
+| ---------------------- | -------------------- | ------------------ | ------------------------------------------------------------------------ |
+| Success                | success              |                    | Worker script executed successfully                                      |
+| Client Disconnected    | clientDisconnected   |                    | HTTP client (i.e. the browser) disconnected before the request completed |
+| Script Threw Exception | scriptThrewException | 1101               | Worker script threw an unhandled Javascript exception                    |
+| ¹ Exceeded Resources   | exceededResources    | 1102, 1027         | Worker script exceeded runtime limits                                    |
+| ² Internal Error       | internalError        |                    | Workers runtime encountered an error                                     |
+
+¹ The Exceeded Resources status may appear when the Worker exceeds a [runtime limit](/about/limits). The most common cause is excessive CPU time, but is also caused by a script exceeding startup time or free tier limits.
```

Comment:
```suggestion
¹ The Exceeded Resources status may appear when the Worker exceeds a [runtime limit](/about/limits). The most common cause is excessive CPU time, but is also caused by a script exceeding startup time or free tier limits.
```

/about/limits is a long page make sure to add a `#` to what part of the section

Reply:
👍